ConsoleApp/WeatherInfo.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Text;
+
+namespace ConsoleApp
+{
+    internal class WeatherInfo
+    {
+        public double Latitude { get; init; }
+        public double Longitude { get; init; }
+        public CurrentWeatherInfo Current_Weather { get; init; }
+        public void WriteInfo()
```

Comment:
instead of writing to console, it's better to have method `string GetInformation()`